### PR TITLE
adding hubot user

### PR DESCRIPTION
**Pull request by rrxtns**

### CODE REVIEW

File: cookbooks/hangops-jobbot/recipes/default.rb [hunk 1]
```diff
@@ -14,6 +14,12 @@
 # limitations under the License.
 #
 
+# Create a Hubot user
+user 'hubot' do
+   action :create #default action
+   group "groupid"
```
Comment (gabinante) [inline cookbooks/hangops-jobbot/recipes/default.rb]: You have any user groups on the host? group is not necessary if you don't want to add it to a user group

Comment (drahamim) [inline cookbooks/hangops-jobbot/recipes/default.rb]: won't it need possible group permissions for access to certain directories?

Comment (gabinante) [inline cookbooks/hangops-jobbot/recipes/default.rb]: yeah, that's fine if the group "groupid" exists. If not, @rrxtns should update the group "groupid" to group "robots" or something.

**Comment by gabinante** [comment]
lgtm

**Pull request merged**